src/sentry/notifications/activity/base.py

```diff
@@ -281,6 +286,20 @@ def update_user_context_from_group(
         user_context.update(context)
         return user_context
 
+    def get_title(self) -> str:
+        return str(self.get_context()["activity_name"])
+
+    def get_dm_text(self) -> str:
+        return str(self.get_context()["text_description"])
```

Comment:
This looks likely to break because not every `ActivityNotification` has `text_description` in `context`. It might make more sense to add `dm_description` to `.get_context()` and use that field in `build_attachment`.